src/Consolonia.Core/Drawing/PixelBufferImplementation/DrawingBoxSymbol.cs

```diff
@@ -31,16 +31,16 @@ public bool Equals(DrawingBoxSymbol other)
 
         [JsonIgnore] public string Text { get; init; }
 
-        [JsonIgnore] public ushort Width { get; } = 1;
+        [JsonIgnore] public ushort Width => 1;
 
-        public bool IsWhiteSpace()
+        public bool NothingToDraw()
```

Comment:
much better name.

Reply:
or perhaps IsEmpty?

Reply:
Honestly I had intention to write a sentence with an explanation there. I bet I need just 1 day to forget whether space is empty or no)